src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -412,3 +412,15 @@
 .monaco-workbench .pane-body.integrated-terminal .terminal-group > .monaco-split-view2.vertical .terminal-drop-overlay.drop-after {
 	top: 50%;
 }
+
+.terminal-command-decoration {
+	transition: opacity 0.5s;
+	transition: all .2s ease-in-out;
+	position: absolute;
```

Comment:
delete

Reply:
this is needed because we set the style of the xterm-decoration to block - probably should be absolute instead?
![Screen Shot 2022-02-10 at 1 02 16 PM](https://user-images.githubusercontent.com/29464607/153478297-f1940ae3-d16b-40ed-a08e-a3c5d28fd473.png)

Reply:
should fix that in xterm